### PR TITLE
Fix front ui

### LINKED ISSUE #1012: Front-ui missing PL icon
On root page of MK.

The login view is missing in the public lab and github icon

image

### PR DESCRIPTION
**Pull request by sashadev-sky**
Fixes #1012 (<=== Add issue number here)

The login view was missing in the public lab icon, and github icon:
image

Re-implemented it and DRYed up code by using the `_login_form` partial instead of rewriting the UI logic.
image

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by sashadev-sky** [comment]
@publiclab/mapknitter-reviewers

**Comment by cesswairimu** [comment]
Wow! This look amazing, love the new fonts and designs :heart: :tada: :tada:. Maybe we should keep the publiclab icon. What do you think?

**Comment by jywarren** [comment]
Hi! This looks good, just re-using the already working code from the upper-right navbar dropdown. Super! Merging this now as I believe this resolves #1010 too?

**Comment by jywarren** [comment]
Thank you Sasha!!! 

(Had trouble sleeping? wow, late night critical fixes FTW! 🙌 )

File: app/views/front_ui/index.html.erb [hunk 1]
```diff
@@ -69,28 +69,8 @@
...
-            <%= button_tag 'Facebook', name: "subaction", value: "facebook" do%>
-              <span class="btn btn-default" style="background-color: #3b5998;width: 60px;">
-                <i class="fab fa-facebook fa-fw"></i>
-              </span>
-            <% end %>
-            </form>
-          <br><br>
+          <p>We strongly prefer that you should
+            <%= render partial: "layouts/login_form" %>
           OR <a href="http://publiclab.org/signup">sign up for a Public Lab account</a>
```
Comment (divyabaid16) [inline app/views/front_ui/index.html.erb]: Hi @sashadev-sky I think this line can be removed now.

**Pull request merged**